src/core/crypto.c

```diff
@@ -1482,7 +1482,9 @@ QuicCryptoProcessTlsCompletion(
             Crypto->TlsState.ReadKey);
 
         //
-        // If we have the read key, we must also have the write key.
+        // Note: Nominally, if we have the write key, we should have the read key
```

Comment:
We can revert this comment, or perhaps expand it to say this is necessary to be able to acknowledge the packets we receive and decrypt.

Reply:
removing it entirely.  Need to review the openssl changes that I have, but if they are accepted in some form, the removal of this should no longer be needed